.htmlvalidate.json

```diff
@@ -5,6 +5,7 @@
 		"no-trailing-whitespace": ["off"],
 		"doctype-style": ["off"],
 		"element-name": ["error", { "whitelist": ["dialog"] }],
-		"missing-doctype": ["off"]
+		"missing-doctype": ["off"],
+		"no-dup-id": ["off"]
```

Comment:
let's fix to have duplicated IDs within our examples instead.

Reply:
I've provided some solution to ensure unique IDs (at least for the iterations for the moment).